NetworkMgr/wg_api.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/local/bin/python3.11
+
+from platform import system
+from subprocess import Popen, PIPE, run, check_output, os
+
+prefix = '/usr/local' if system() == 'FreeBSD' else sys.prefix
+wgconfigpath = prefix + '/etc/wireguard/'
+
+def wg_service_state():
+    run('service wireguard status', shell=True)
+
+def wireguardDictionary():
```

Comment:
I have not yet rolled out the guide on how the Python code should be written, but it should adhere to the PEP8 standard. Methods should be `wireguard_dictionary` instead of `wireguardDictionary`, same with variables.

I know there is old code that doesn't meet that standard yet, but new code should.